spec/fixtures/database-issue84.bib

```diff
@@ -0,0 +1,50 @@
+
+@article{baran_distributed_1964,
+	title = {On distributed communications networks},
+	volume = {12},
+	issn = {0096-1965},
+	number = {1},
+	journal = {IEEE transactions on Communications Systems},
+	author = {Baran, Paul},
+	year = {1964},
+	pages = {1--9},
+	file = {Bar64.pdf:/Users/reza/Zotero/storage/9T733JXT/Bar64.pdf:application/pdf}
+}
+
+@article{lukasik_why_2011,
+	title = {Why the {Arpanet} {Was} {Built}},
+	volume = {33},
+	issn = {1058-6180},
+	url = {http://ieeexplore.ieee.org/document/5432117/},
+	doi = {10.1109/MAHC.2010.11},
+	language = {en},
+	number = {3},
+	urldate = {2019-02-20},
+	journal = {IEEE Annals of the History of Computing},
+	author = {Lukasik, Stephen},
+	month = mar,
+	year = {2011},
+	pages = {4--21},
+	file = {Lukasik - 2011 - Why the Arpanet Was Built.pdf:/Users/reza/Zotero/storage/I4F9Y8VI/Lukasik - 2011 - Why the Arpanet Was Built.pdf:application/pdf}
+}
+
+@inproceedings{roberts_arpanet_1986,
+	title = {The {Arpanet} and computer networks},
+	isbn = {0-89791-176-8},
+	publisher = {ACM},
+	author = {Roberts, Larry},
+	year = {1986},
+	pages = {51--58},
+	file = {p51-roberts.pdf:/Users/reza/Zotero/storage/IKUSGSA8/p51-roberts.pdf:application/pdf}
+}
+
+@article{dommering_internet:_2015,
+	title = {The {Internet}: a global free space with limited state control},
+	shorttitle = {The {Internet}},
+	url = {https://dare.uva.nl/search?identifier=8672fb24-0825-4bdb-98ac-989e370446ae},
+	language = {en},
+	urldate = {2019-02-20},
+	author = {Dommering, E. and van Ginkel, B. and de Goede, M. and Koops, B. J. and Plooij-van Gorsel, E. and Verrijn Stuart, H. and Smallenbroek, J.},
+	year = {2015},
+	file = {THE INTERNET A GLOBAL FREE SPACE WITH LIMITED STAT.pdf:/Users/reza/Zotero/storage/MQJYNM6X/THE INTERNET A GLOBAL FREE SPACE WITH LIMITED STAT.pdf:application/pdf}
```

Comment:
Remove `/Users/reza/Zotero/storage/MQJYNM6X/THE INTERNET A GLOBAL FREE SPACE WITH LIMITED STAT.pdf`

Reply:
Fixed